AsyncDisplayKitTests/ASTableViewTests.m

```diff
@@ -290,4 +294,138 @@ - (void)triggerSizeChangeAndAssertRelayoutAllRowsForTableView:(ASTableView *)tab
   }];
 }
 
+- (void)testRelayoutVisibleRowsWhenEditingModeIsChanged
```

Comment:
Love the new tests — thanks!  Code coverage in ASDK continues to increase, and I think more of it is due to your tests than anyone else.